engine/graphics/src/test/test_graphics.cpp

```diff
@@ -570,23 +548,26 @@ TEST_F(dmGraphicsTest, TestComputeProgram)
         "void main() {\n"
         "}\n";
 
+    dmGraphics::ShaderDesc::ResourceTypeInfo types[1] = {};
+    FillShaderResourceTypeInfo(&types[0], "my_uniform", dmGraphics::ShaderDesc::SHADER_TYPE_VEC4);
+
     dmGraphics::ShaderDesc::ResourceBinding uniform = {};
-    FillResourceBinding(&uniform, "my_uniform", dmGraphics::ShaderDesc::SHADER_TYPE_VEC4);
+    FillResourceBindingTypeIndex(&uniform, "my_uniform", 0, 0);
```

Comment:
For uniform buffers, we need to split the resource binding into the binding and the type. We can perhaps simplify the helper functions later